pkg/project/registry/proxy/proxy_test.go

```diff
@@ -70,24 +69,10 @@ func TestCreateProjectBadObject(t *testing.T) {
 	}
 }
 
-func TestCreateProjectMissingID(t *testing.T) {
```

Comment:
The test is invalid because now that this follows the upstream patterns for strategy a name is generated of omitted by client.

Sent from my iPhone

> On Apr 6, 2015, at 4:04 PM, David Eads <notifications@github.com> wrote:
> 
> In pkg/project/registry/proxy/proxy_test.go:
> 
> > @@ -70,24 +69,10 @@ func TestCreateProjectBadObject(t *testing.T) {
> >  	}
> >  }
> >  
> > -func TestCreateProjectMissingID(t *testing.T) {
> Why is this test now invalid? I don't see a replacement in another package in this pull.
> 
> —
> Reply to this email directly or view it on GitHub.
>